### DISCUSSION TITLE
PiVPN no longer maintained

### DISCUSSION THREAD

**Discussion by LinuxSBC** [tteck/Proxmox#2748] [General]
PiVPN's maintainer has decided to stop maintaining the project: https://github.com/pivpn/pivpn/releases/tag/v4.6.0

Since the Wireguard LXC uses PiVPN, it would be best to switch to a maintained project (maybe [wg-easy](https://github.com/wg-easy/wg-easy) if being a Docker container is not an issue).

**Comment by tteck** [comment]
Why not just a `apt install wireguard` on Debian 12 as the server?

https://wiki.debian.org/WireGuard

**Reply by LinuxSBC** [reply]
Personally, I prefer having the script to just run `pivpn add` or similar rather than having to manually generate the keypairs and create the client and server conf files. However, that's not very significant. If you don't want to add support for a different set of scripts or UI, that's fine; I don't want to ask you to do something that you don't want to do, and it's not hard to do it manually or use something like wg-easy on top of the Docker LXC.

**Reply by iz8mbw** [reply]
I also prefer to have a LXC like is now done with PiVPN.
[wg-easy](https://github.com/wg-easy/wg-easy) seems a valid replacement.
Hope it will implemented.
Thanks!

**Comment by fanaticDavid** [comment]
With PiVPN no longer being maintained, I combined the suggestions above:

1. I created an LXC container based on Debian 12.
2. I followed the WireGuard wiki above to install WireGuard, although I skipped any client configuration.
3. I followed [this wiki](https://github.com/wg-easy/wg-easy/wiki/Using-WireGuard-Easy-without-Docker) for installing wg-easy without Docker.
4. I used the wg-easy web UI to create client configurations.

**Comment by iwconfig** [comment]
Still maintained --> https://github.com/pivpn/pivpn/releases/tag/v4.6.1
